task.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListner
+const number1 = Number(prompt('First number'));
+const number2 = Number(prompt('Second number'));
+
+const answerplus = (number1 + number2)
+const answerminus = (number1 - number2)
+const answermultiply = (number1 * number2)
+const answerdivide = (number1 / number2)
+
+alert (`${(number1+" + "+number2+" = "+ answerplus)} 
```

Comment:
looks like You lose template strings idea(
```suggestion
alert (`${number1} + ${number2} = ${answerplus}`)
```
OR
```suggestion
alert (number1 + ' + ' + number2 + ' = ' + answerplus)
```